File: 1.FDP/2° Competencia/Java/Basicos/Ejercicio2.java
import java.util.Scanner;	

public class Ejercicio2 {
	public static void main (String[] args){
		Scanner scan= new Scanner(System.in);
		//
		System.out.println("Calcular el area de un rectagulo.\n");
		double Base, Altura, Area;
		System.out.println("Base: ");
		Base= scan.nextDouble();
		System.out.println("Altura: ");
		Altura= scan.nextDouble();
		Area= Base*Altura;
		System.out.println("Area= " + Area);
		//
	}
}

File: 1.FDP/2° Competencia/C++/Basicos/Ejercicio2.cpp
#include <iostream>
using namespace std;

int main(){
	cout << "Calcular el area de un rectangulo.\n";
	double Altura, Base, Area;
	cout << "Base: ";
	cin >> Base;
	cout << "Altura: ";
	cin >> Altura;
	Area= Altura*Base;
	cout << "Area = " << Area << "\n";
	return 0;
}
